demos/python_demos/smart_library_demo/README.md

```diff
@@ -0,0 +1,136 @@
+# Smart Library Demo
+
+The applicattion is a demo of automated “smart library”. 
+It involves the registration of the reader; authorization of the reader through face recognition; 
+receiving and returning books by recognizing QR codes generated for each book in the library. 
+The following pretrained models can be used:
+
+* `face-detection-retail-0004`, to detect faces and predict their bounding boxes;
+* `landmarks-regression-retail-0009`, to predict face keypoints;
+* `face-reidentification-retail-0095`, to recognize readers.
+
+For more information about the pre-trained models, refer to the [model documentation](../../../models/intel/index.md).
+
+### How it works
+
+The application is started from command line.
+It reads video stream frame-by-frame from a web-camera device and performs independent analysis
+of each frame. To make predictions the application uses 3 models. An input frame is processed by
+the face detection model to detect bounding boxes. Then face keypoints
+are predicted by the facial landmarks regression model. Keypoints are using
+to align the face and match it with face in data base. 
+To register in library press `r` on keyboard. Once reader has beeen registered, he will be autothorized 
+through face recognition. Registration allows to receiving and returning books by recognizing QR codes for 
+each book. To make book recognition press `b` on keyboard. Also application provides some 
+extra statictics in console, like list of registered readers, full list of books in the library,
+history of borrowing books. To change information in console press `f` on keyboard. To exit press `q`.
+
+### Installation and dependencies
+
+The demo depends on:
+- OpenVINO toolkit (2019R3 or newer)
+- Python (any of 2.7+ or 3.4+, which is supported by OpenVINO)
+- OpenCV (>=3.4.0)
+
```

Comment:
We don't need to specify this. OMZ is supposed to work with OpenVINO and OpenCV of versions it is shipped with.